fetch.js

```diff
@@ -6,12 +6,18 @@ require("dotenv").config();
 const GITHUB_TOKEN = process.env.REACT_APP_GITHUB_TOKEN;
 const GITHUB_USERNAME = process.env.GITHUB_USERNAME;
 const USE_GITHUB_DATA = process.env.USE_GITHUB_DATA;
+const USE_MEDIUM_DATA = process.env.USE_MEDIUM_DATA;
+const MEDIUM_USERNAME = process.env.MEDIUM_USERNAME;
```

Comment:
Imo rather than two variables; it would be better to check if `MEDIUM_USERNAME` is defined, if defined fetch data else don't.

Reply:
done: https://github.com/saadpasta/developerFolio/pull/445/commits/e2d654f06084f46b5aa8139cd034300cbcb17daf

Reply:
What I meant was to not read for the variable `USE_MEDIUM_DATA` and use `MEDIUM_USERNAME` to check whether or not to create the file.

Reply:
I forgot to commit fetch.js, sorry. See https://github.com/saadpasta/developerFolio/pull/445/commits/d94a992ba690d55928b01e58739d290893b73f3b.

I've removed USE_MEDIUM_DATA, since there is no point to keep it if we are only checking for MEDIUM_USERNAME